demos/GMW_demo/1-out-of-n/IO.js

```diff
@@ -0,0 +1,59 @@
+/*
+ *  the IO object to be used for OT
+ *  to pass as the IO parameter for 1-out-of-N.
+ *
+ */
+// IO send
+const give = function ( op_id, jiff,session_id, tag, msg) {
+  /* give a message */
+  const createMSGObj=function (msg,op_id,tag,jiff) {
+    var four_opts={};
+    four_opts['opts']=msg;
+    four_opts['sender_id']=jiff.id;
+    four_opts['op_id']=op_id;
+    four_opts['tag']=tag;
+    return four_opts;
+  };
+  var temp=(op_id.split(':')[3]).split('-');
```

Comment:
If this string manipulation happens in more than one place (e.g. maybe in listen('OT', ...)) we should have a function that manages it and can be called from both places.

Reply:
could return `[to_id, from_id]`